File: Notes/conditionals.py
"""number = -12

if abs(number) < 10:
    print(f"{number} is a single digit number")
elif abs(number) < 100:
    print(f"{number} is a 2 digit number")
else:
    print("Your number is too big")"""

name = input("What is your name:").strip().capitalize()

if not name == "Vienna" and not name == "Andrew":
    print("I love your hair. How often do you dye it?")
    often = input("In months how often do you dye your hair: ")
    if int(often) < 3:
        print("Wow that is a lot of work!")
    else:
        print("That is super cool!")
else:
    print(f"Hello {name}!")

File: Notes/conditionals.c
// AL Conditionals
#include <stdio.h>
#include <string.h>

int main(){
    int grade = 97;

    if (grade >= 90){
        if (grade > 93){}
        printf("You have a %d percent. That is an A!\n", grade);
    }else{
        printf("You have a %d percent. That is an A-!\n", grade);
    }
    }else if (grade >= 80){
        printf("You have a %d percent. That is a B!\n", grade);
    }else if (grade >= 70){
        printf("You have a %d percent. That is a C.\n", grade);
    }else if (grade >= 60){
        printf("You ahve a %d percent. That is a D.\n", grade);
    }else{
        printf("Oh no! You have a %d percent, that is failing!\n", grade);
    };

    int num = 4;

    if (num%2 == 0 && (num <10 && num > -10)){
        printf("%d is a single digit even number.\n", num);
    }else if (num%2 != 0 && (num <10 && num > -10)){
        printf("%d is a single digit odd number!\n", num);
    }else{
        printf("%d is not a single digit number!\n", num);
    }

    char name[] = "Eric";

    if(strcmp(name, "Cora") == 0){
        printf("Welcome Admin!\n");
    }else{
        printf("Hello %s\n", name);
    }
        
    return 0;
}